<?xml version="1.0"?>
<launch>

    <arg name="rviz" default="true" />
    <arg name="project" default="multi_sensor_mapping"/>

    <!-- Lidar IMU Mapper -->
    <node pkg="$(arg project)" type="feature_extractor_viewer" name="feature_extractor_viewer" output="screen" respawn="false">
        <param name="bag_path" value="/home/csc105/dataset/shs_backpack.bag"/>
        <param name="lidar_topic" value="/rslidar_points"/>
    </node>


    <!--- Rviz -->
    <group if="$(arg rviz)">
      <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find multi_sensor_mapping)/rviz/feature_extractor_viewer.rviz" />
    </group>
</launch>
